src/daemon/daemon.cpp

```diff
@@ -135,14 +143,15 @@ bool t_daemon::run(bool interactive)
   {
     if (!mp_internals->core.run())
       return false;
-    mp_internals->rpc.run();
 
-    std::unique_ptr<daemonize::t_command_server> rpc_commands;
+    for(const auto& rpc: mp_internals->rpcs)
+      rpc->run();
```

Comment:
Logically, it seems odd to call run on a const object. I'd make it non const, it's less surprising I think ?

Reply:
Sounds good, yeah agreed I'll switch those instances of it to remove that.